### ISSUE TITLE
[BLUEJ-1358] No way to read output in terminal via keyboard navigation

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/BlueJ-Greenfoot#1331]
If you want to read stdout from the terminal there is no way to do this with keyboard navigation.  The pane can't be tabbed into in order to navigate with the keyboard to the output line to be read out.  The pane should be made focusable but non-editable, rather than non-focusable.

---

**Issue metadata**

- Issue type: Bug
- Priority: Medium
- Fix versions: 5.0.3
